src/notifications/settings/initialization.ts

```diff
@@ -121,42 +124,27 @@ export const _prepareSubscriptionQueueAndCreateInitialSettings = (
       const oldSettingsEntry = newSettings[alreadySavedEntry.index];
       const updatedSettingsEntry = {
         ...oldSettingsEntry,
+        enabled: true,
         topics: DEFAULT_ENABLED_TOPIC_SETTINGS,
         type: entry.relationship,
         successfullyFinishedInitialSubscription: false,
-        oldType: alreadySavedEntry.settings.type,
```

Comment:
`oldType` was used to keep track of scenarios when a watched wallet got imported fully (you would have to unsubscribe the watch wallet and subscribe as fully imported). now, we just need to update the notification settings as expected (new relationship as "owned" and default of enabled for an imported wallet)